learn/core_concepts/indexes.md

```diff
@@ -48,55 +35,91 @@ The `uid` is set at [index creation time](/reference/api/indexes.md#create-an-in
 
-Words added to the [stop words list](/reference/api/stop_words.md) will be ignored during search. In addition to improving relevancy, designating common words as stop words also greatly improves performance.
+[Learn more about sorting](/learn/advanced/sorting.md)
 
-For example, suppose you want to search for `the great gatsby`. You would prefer to receive documents containing the terms `great gatsby`, rather than documents containing the terms `the great`, or just `the`. In this case, adding `the` to the stop word list would improve performance and make search results more relevant.
+### Stop words
+
+Your dataset may contain words you want to ignore during search. These may include words that don't add semantic value or context or are too frequent (e.g., `the` or `of` in English). You can add such words to the [stop words list](/reference/api/stop_words.md) and Meilisearch will ignore them during search. In addition to improving relevancy, designating common words as stop words also greatly improves performance.
+
+You can create your list of stop words for your index using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update stop words endpoint](/reference/api/stop_words.md#update-stop-words).
 
 [Learn more about stop words](/reference/api/stop_words.md)
 
-## Field properties
+### Displayed and searchable attributes
+
+By default, every document field is searchable and displayed in response to search queries. However, you can choose to set some fields as non-searchable, non-displayed, or both.
+
+You can update these field attributes using the [update settings endpoint](/reference/api/settings.md#update-settings), or the respective update endpoints for [displayed attributes](/reference/api/displayed_attributes.md#update-displayed-attributes), and [searchable attributes](/reference/api/searchable_attributes.md#update-searchable-attributes).
+
+[Learn more about displayed and searchable attributes](/learn/configuration/displayed_searchable_attributes.md)
 
-By default, every document field is searchable and returned on search queries.
+### Typo tolerance
 
-Fields can have either or both or none of the following properties that can be modified in the [settings](/reference/api/settings.md):
+Typo tolerance is a built-in feature that helps you find relevant results even when your search queries contain spelling mistakes or typos, e.g., typing `chickne` instead of `chicken`. The setting allows you to configure the following for an index:
```

Comment:
```suggestion
Typo tolerance is a built-in feature that helps you find relevant results even when your search queries contain spelling mistakes or typos, e.g., typing `chickne` instead of `chicken`. This setting allows you to do the following for your index:
```